default/stringtables/en.txt

```diff
@@ -19542,27 +19573,68 @@ Willow'''
 
 # Newline separated list of mineral-based ship names
 SHIP_NAME_STONE
-'''Amphibolite
+'''Adularia
+Agate
+Amber
+Amphibolite
 Andesite
+Anhydrite
+Asphalt
 Basalt
+Borax
+Chalcedony
 Chalk
+Chrysoberyl
 Clay
+Concrete
+Corundum
+Diamond
+Diopside
 Dolomite
+Emerald
+Feldspar
+Ferrite
 Flint
 Gabbro
 Gneiss
 Granite
+Haematite
+Hornblende
+Jasper
+Jett
 Komatite
+Lapislazuli
 Limestone
+Malachite
 Marble
+Melilith
+Menhir
+Nugget
+Obelisk
```

Comment:
not a mineral, I think?

Reply:
but can consist of minerals and is stone-related

Reply:
lots of things can consist of stone. the theme of this list kinds of stone or minerals, not stuff made from it.

Reply:
seeing how they are mixed together in the end anyway, i fail to see the point here. but whatever, they can go to the 'others' group